graphql/resolvers.go

```diff
@@ -115,16 +115,15 @@ func (r *queryResolver) Projects(ctx context.Context) (*Projects, error) {
 			Owner:       p.Owner,
 		}
 
+		if isFavorite := util.StringSliceContains(favoriteIds, p.Identifier); isFavorite {
```

Comment:
you don't need to declare isFavorite here. 

I also wouldn't declare favoriteIds as its own variable, and just reference usr.FavoriteProjects. Extra variables impede readability.

Reply:
`isFavorite` does not _need_ to be declared here but it also does not hurt and adds semantic value imo

Reply:
I'm pretty convinced that the determent to readability of declaring extra variables is not worth any potential benefit of added semantics/annotations.

Reply:
i think `isFavorite` increases readability because it reads like english. `if isFavorite .... { do this }`

Reply:
As a similar example, PEP8 disagrees. They argue that `if greeting is True` is worse than `if greeting == true` or `if greeting`, even though the first is the most English. The `if isFavorite` requires reading to the end of the line, and is definitely not idiomatic go. I had to read the line a couple times before I got it. I'm used to reading `if <expression>` in many programming languages, not `if <thing> := <expression>; <thing>`, which is more things for my eyes to parse.

The usual way to use this idiom in go is:
```
if err := <function that returns error>; err != nil {
// handle error
}
```

And somewhat less usually, something like:
```
if val, ok := <thing with two return values>; ok {
// handle val, which only exists with ok is true
}
```

It's meant for creating a branch where one does not already exist. In this case though there's an expression that already evaluates to a boolean. The advantage here is that the code is somewhat more self-documenting, but I find it hard to read, and it is probably better in a comment.

Reply:
```
if itMattersThisMuch := GetStrongOpinions(); itMattersThisMuch {
  then by all means I shall change it :)
}
```